src/allmydata/scripts/runner.py

```diff
@@ -177,9 +174,8 @@ def _maybe_enable_eliot_logging(options, reactor):
     return options
 
 def run():
-    # TODO(3035): Remove tox-check when error becomes a warning
-    if 'TOX_ENV_NAME' not in os.environ:
-        assert sys.version_info < (3,), u"Tahoe-LAFS does not run under Python 3. Please use Python 2.7.x."
+    if not six.PY2:
+        warnings.warn("Support for Python 3 is experimental. Use at your own risk.")
```

Comment:
Let's strengthen this even more.  Python 3 support is not just experimental right now.  It is **in-progress** - known to be incomplete.